packages/config/src/projects/arbitrum.ts

```diff
@@ -88,12 +93,24 @@ export const arbitrum: Project = {
         name: 'Optimistic Rollup',
       },
       stateCorrectness: {
-        ...STATE_CORRECTNESS.FRAUD_PROOFS,
+        name: 'Fraud proofs ensure state correctness',
+        description:
+          'After some period of time, the published state root is assumed to be correct. For a certain time period, usually one week one of the whitelisted actors can submit a fraud proof that shows that the state was incorrect.',
+        risks: [
+          {
+            category: 'Funds can be stolen if',
+            text: 'there is no one that checks the published state. Fraud proofs assume at least one honest and able validator.',
```

Comment:
This risk should mention the fact that this requires being whitelisted. Potentially could be marked as critical.